src/VisualStudio/TestUtilities/Proxy/RemotingProxy.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+using Roslyn.VisualStudio.Test.Utilities.Remoting;
+
+namespace Roslyn.VisualStudio.Test.Utilities.Proxy
+{
+    internal static class RemotingProxy
```

Comment:
Why were these moved here? They were (and should be) only called from a single location.